docs/basics.rst

```diff
@@ -171,7 +171,7 @@ From the above, you may have noticed that each test gets a series id
 like ``s24`` and a test id like ``41``. You can use these id's to
 reference tests or suites of tests to get their status, results, and
 logs through the pavilion interface. The ID's are unique for a given
-Pavilion `working\_directory <config.html#working_dir>`__, but they will
+Pavilion `working\_directory <config.html#working-dir>`__, but they will
```

Comment:
Shouldn't this be `config.rst`? I don't see a `config.html` in the `docs` directory.